### PR TITLE
added regen-hambach-1

### PR DESCRIPTION
**Pull request by dpdanpittman**
details and genesis.json for hambach devnet

### CODE REVIEW

File: hambach-devnet/api-nodes.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+https://137.184.24.185:1317
```
Comment (ryanchristo) [inline hambach-devnet/api-nodes.txt:1]: I'm not getting a response from this url. Are you sure this is the correct url and do you have api enabled?

File: hambach-devnet/genesis.json [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+        "sig_verify_cost_secp256k1": "1000"
+      },
+      "accounts": [
+        {
+          "@type": "/cosmos.auth.v1beta1.BaseAccount",
+          "address": "regen1hjr5ukxfxqlcef363an2snd0wgsnkte0xh8wmr",
+          "pub_key": null,
+          "account_number": "0",
+          "sequence": "0"
+        },
```
Comment (ryanchristo) [inline hambach-devnet/genesis.json:40]: This is an additional account when compared to regen-redwood. Can you clarify the accounts being used?

File: hambach-devnet/genesis.json [hunk 3]
```diff
@@ -0,0 +1,360 @@
...
+        {
+          "address": "regen1pxag6wu3jz885yr9t8gkuvc4ktxxt2la7xm4x3",
+          "coins": [
+            {
+              "denom": "uregen",
+              "amount": "1000000"
+            }
+          ]
+        },
```
Comment (ryanchristo) [inline hambach-devnet/genesis.json:99]: Is there a reason the starting balance is different for this account?

File: hambach-devnet/genesis.json [hunk 4]
```diff
@@ -0,0 +1,360 @@
...
+    },
+    "ecocredit": {
+      "params": {
+        "credit_class_fee": [
+          {
+            "denom": "uregen",
+            "amount": "10000"
+          }
+        ],
+        "allowed_class_designers": [],
```
Comment (ryanchristo) [inline hambach-devnet/genesis.json:156]: We are discussing a better name for this parameter. This is correct but might change in the near future in which case we will need to update the genesis file and restart the testnet.

File: hambach-devnet/genesis.json [hunk 5]
```diff
@@ -0,0 +1,360 @@
...
+        "min_deposit": [
+          {
+            "denom": "uregen",
+            "amount": "10000000"
+          }
+        ],
+        "max_deposit_period": "172800s"
+      },
+      "voting_params": {
+        "voting_period": "3600s"
```
Comment (ryanchristo) [inline hambach-devnet/genesis.json:259]: We're using `172800s` in redwood. Should we be consistent with the voting period on both testnets?

File: hambach-devnet/peer-nodes.txt [hunk 6]
```diff
@@ -0,0 +1 @@
+4f5c0be7705bf4acb5b99dcaf93190059ac283a1@137.184.24.185:26656\
```
Comment (ryanchristo) [inline hambach-devnet/peer-nodes.txt:1]: I'm not getting a response from this url when updating the port to 26657. Are you sure this is the right url?

File: hambach-devnet/rpc-nodes.txt [hunk 7]
```diff
@@ -0,0 +1 @@
+http://137.184.24.185:26657
```
Comment (ryanchristo) [inline hambach-devnet/rpc-nodes.txt:1]: No response from this url. Is this correct? See http://128.199.249.31:26657/ for example.

**Review by ryanchristo — COMMENTED** [review]
Looking good!

It looks like the url has changed. I'm not receiving a response when attempting to check the status or access the api. Also, I'm wondering if we should be more consistent between the two testnets. The voting period and genesis accounts are different.

File: hambach-devnet/api-nodes.txt (hunk 1 above)
Comment (dpdanpittman) [reply hambach-devnet/api-nodes.txt:1]: resolved

File: hambach-devnet/genesis.json (hunk 5 above)
Comment (dpdanpittman) [reply hambach-devnet/genesis.json:259]: We decided to make the voting periods really short on this to speed up changes

File: hambach-devnet/peer-nodes.txt (hunk 6 above)
Comment (dpdanpittman) [reply hambach-devnet/peer-nodes.txt:1]: resolved

File: hambach-devnet/genesis.json (hunk 2 above)
Comment (dpdanpittman) [reply hambach-devnet/genesis.json:40]: Will clarify in network ops channel

**Pull request merged**